.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,16 @@
 
 ## This introduces a breaking change
 
-- [ ] Yes
-- [ ] No
+- Yes
+- No
```

Comment:
Hmm... that change is already in the master branch (392118c4e75b509364a2c52a92f0dd5e7eda5dc9 in #29).

Can you rebase?